samtranslator/feature_toggle/feature_toggle.py

```diff
@@ -18,8 +18,22 @@ class FeatureToggle:
     SAM is executing or not.
     """
 
-    def __init__(self, config_provider):
+    def __init__(self, config_provider, stage="beta", account_id=None, region="default"):
         self.feature_config = config_provider.config
+        self.stage = stage
+        self.account_id = account_id
+        self.region = region
+
+    def is_enabled(self, feature_name):
```

Comment:
Why is this method needed? e.g. instead of moving the `if` to the actual call?

Reply:
I renamed `is_enabled_for_account_in_region` to this one, removed `is_enabled_for_stage_in_region`. What `is_enabled_for_stage_in_region` does is already done in `is_enabled_for_account_in_region`.

I also made `stage`, `account_id`, `region` required when instanciating `FeatureToggle`

Reply:
Nice! Turned out really nice! 👍